radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/PeerEvent.java

```diff
@@ -85,23 +83,6 @@ private PeerConnected(PeerChannel channel) {
 		public PeerChannel getChannel() {
 			return this.channel;
 		}
-
-		@Override
-		public boolean equals(Object o) {
-			if (this == o) {
-				return true;
-			}
-			if (o == null || getClass() != o.getClass()) {
-				return false;
-			}
-			final var that = (PeerConnected) o;
-			return Objects.equals(channel, that.channel);
```

Comment:
equals/hashCodes using the channel didn't really make sense in these events classes